src/test/java/com/flightstats/hub/cluster/DistributedAsynchronousLockRunnerTest.java

```diff
@@ -0,0 +1,114 @@
+package com.flightstats.hub.cluster;
+
+import com.flightstats.hub.test.Integration;
+import org.apache.curator.framework.CuratorFramework;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static com.google.common.collect.Lists.newArrayList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class DistributedAsynchronousLockRunnerTest {
```

Comment:
Same here. can we shorten the corresponding class name to DistributedAsyncLockTest?